docs/resources/samples/combine-worksheets-into-single-workbook.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Combine workbooks into a single workbook'
+description: 'Learn how to use Office Scripts and Power Automate to create merge worksheets from other workbooks into a single workbook.'
+ms.date: 09/03/2021
+localization_priority: Normal
+---
+
+# Combine worksheets into a single workbook
+
+This sample shows how to pull data from multiple workbooks into a single, centralized workbook. It uses two scripts: one to retrieve information from a workbook and another to create new worksheets with that information. It combines the scripts in a Power Automate flow that acts on an entire OneDrive folder.
+
+> [!IMPORTANT]
+> This sample only copies the values from the other workbooks. It does not preserve formatting, charts, tables, or other objects.
+
+## Scenario
+
+1. Create a new Excel file in your OneDrive and add two scripts from this sample to it.
+1. Create a folder in your OneDrive and add at least one workbook with data to it.
+1. The flow you'll build gets all the files in a specified folder full of Excel workbooks.
```

Comment:
"full of Excel workbooks" is ambiguous to me. It seems in conflict with the previous step. Why is the folder full of Excel workbooks? Does this mean that the flow only retrieves Excel workbooks but other types of files could be in the folder? Does it mean that the folder *could* be full of Excel workbooks?

Reply:
The scenario is merging a bunch of workbooks into one. It can just be one other workbook. I reworded this whole section to better convey the user actions.